test.disabled/ejabberd_tests/tests/mod_global_distrib_SUITE.erl

```diff
@@ -227,23 +228,47 @@ end_per_testcase(CaseName, Config) ->
 generic_end_per_testcase(CaseName, Config) ->
     lists:foreach(
       fun({NodeName, _, _}) ->
+              %% TODO: Enable refresher only for specific test cases,
```

Comment:
I'm not sure, but I feel like if we have to make such exclusion, it means we are not testing the real program. In reality the refresher will always be started and never paused.

Reply:
Theoretically yes but some test cases are supposed to verify correctness of on-demand connection establishment. That's why I haven't introduced it here but IMO should be a subject of separate discussion.